src/backend/error.rs

```diff
@@ -5,51 +5,49 @@ use dpp::ProtocolError;
 use rs_dapi_client::DapiClientError;
 
 use crate::backend::{
-    error::Error::{ParsingError, SdkError, WalletError},
+    error::Error::{Parsing, Sdk, Wallet},
     insight::InsightError,
     wallet,
 };
 
 #[derive(Debug, thiserror::Error)]
 pub(crate) enum Error {
```

Comment:
I see this type wasn't added in this PR, but anyways why it exists?

The common denominator for errors was `String` because the sole purpose of errors was to be shown on UI, each backend module exported it's own error type and that was fine, because the next step was to prepare UI event for display task results, so it was a string, why this `error` module even needed?